### PR TITLE
fix(alertGroup): fix a11y

### LINKED ISSUE #1562: role="status" on alert-group component causing a11y violations
Should we just increase the a11y threshold to account for these, or is there another preferred approach?

Screen Shot 2019-03-12 at 4 28 14 PM
Screen Shot 2019-03-12 at 4 27 57 PM

cc @jgiardino @seanforyou23

### PR DESCRIPTION
**Pull request by christiemolloy**
fix #1562

### CODE REVIEW

**Comment by christiemolloy** [comment]
@srambach still getting this error:

1 error found in: /components/AlertGroup/examples-full/
1) <li> elements must be contained in a <ul> or <ol> -- https://dequeuniversity.com/rules/axe/3.1/listitem -- impact: serious
      
-------html violation instances-------
<li class="pf-c-alert-group__item">
<li class="pf-c-alert-group__item">
<li class="pf-c-alert-group__item">

File: src/patternfly/demos/Alert/examples/alert-toast-example.hbs [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {{> page-demo-default}}
-{{#> alert-group alert-group--attribute='role="status"' alert-group--modifier="pf-m-toast"}}
+{{#> alert-group alert-group--modifier="pf-m-toast"}}
```
Comment (jgiardino) [inline src/patternfly/demos/Alert/examples/alert-toast-example.hbs]: So weird! I see that you removed role="status", but it's still showing up in this example:

```
<ul class="pf-c-alert-group pf-m-toast" role="status">
  <li class="pf-c-alert-group__item">
```

**Comment by christiemolloy** [comment]
Ah looks like it was in two examples! should be good now - @srambach @jgiardino

File: src/patternfly/components/AlertGroup/docs/alert-group.md [hunk 2]
```diff
@@ -6,7 +6,6 @@
 
 | Attribute | Applied To | Outcome |
 | -- | -- | -- |
-| `role="status"` | `.pf-c-alert-group` |  Communicates contents of the alert message to the user. **Required** |
```
Comment (mcoker) [inline src/patternfly/components/AlertGroup/docs/alert-group.md]: Can you remove this whole a11y section?

Screen Shot 2019-03-13 at 1 54 37 PM

**Comment by srambach** [comment]
Latest changes fix the Alert and AlertGroup errors I saw before...but the total increased because it tacked more onto the end. Reran it, now it's down to 18. I am fine with merging (with @mcoker's suggestion above) because the errors this intended to fix seem to be fixed.

File: src/patternfly/components/AlertGroup/docs/alert-group.md (hunk 2 above)
Comment (christiemolloy) [reply src/patternfly/components/AlertGroup/docs/alert-group.md]: Yep

**Comment by christiemolloy** [comment]
@srambach if you take a look now I merged the table changes, we are down to 15 violations and a threshold of 16

**Comment by mcoker** [comment]
Should we have the threshold match the current number of violations?

**Pull request merged**